lib/web_ui/lib/platform_dispatcher.dart

```diff
@@ -80,7 +80,7 @@ abstract class PlatformDispatcher {
 
   void scheduleFrame();
 
-  void render(Scene scene, [FlutterView view]);
+  Future<void> render(Scene scene, [FlutterView view]);
```

Comment:
It looks like the mobile version no longer has this method.

Reply:
Agh shoot, forgot to remove it from our interface 🤦